app/src/main/java/com/example/taskyy/ui/events/ReminderEvent.kt

```diff
@@ -3,4 +3,14 @@ package com.example.taskyy.ui.events
 interface ReminderEvent {
 
     data object SetDateString : ReminderEvent
+    data class AlarmTypeDropDownSelected(var isAlarmSelectionExpanded: Boolean) : ReminderEvent
+    data class TimePickerSelected(var timePickerSelected: Boolean) : ReminderEvent
+    data class AlarmTimeTextSelected(var alarmTimeText: String) : ReminderEvent
```

Comment:
Could make sense to make use of an enum class here to have a set of distinct options for these reminders:
```
enum class ReminderType {
    10_MINUTES_BEFORE,
    30_MINUTES_BEFORE,
    ...
}
```
That also makes it easier to check what was passed in the VM, since a when-expression forces you to go through all the different options